modules/ROOT/pages/mule-high-availability-ha-clusters.adoc

```diff
@@ -217,7 +235,7 @@ Remember that application-level configuration overrides container-level configur
 [WARNING]
 In cases of high load with endpoints that do not support load balancing, applying the performance profile may degrade performance. If you are using a File-based transport with an asynchronous processing strategy, JMS topics, multicasting, or HTTP connectors without a load balancer, the high volume of messages entering a single node can cause bottlenecks, and thus it can be better for performance to turn off the performance profile for these applications.
 
-You can also choose to define a minimum number of machines required in a cluster for it to remain in an operational state. This grants you a consistency improvement. See  xref:creating-and-managing-a-cluster-manually.adoc#quorum-management[Quorum Management].
+You can also choose to define a minimum number of machines required in a cluster for it to remain in an operational state. This grants you a consistency improvement.
```

Comment:
for it -> for the cluster

This grants you... // Are we saying "This configuration improves consistency" ? If so, I'm not sure what sort of consistency we're talking about.